source/dub/recipe/json.d

```diff
@@ -251,10 +252,14 @@ private Json toJson(in ref BuildSettingsTemplate bs)
 {
 	auto ret = Json.emptyObject;
 	if( bs.dependencies !is null ){
-		auto deps = Json.emptyObject;
-		foreach( pack, d; bs.dependencies )
-			deps[pack] = serializeToJson(d);
-		ret["dependencies"] = deps;
+		Json[string] deps;
+		foreach( pack, d; bs.dependencies ){
+			string key = d.platform ? "-"~d.platform : "";
+			deps.require(key, Json.emptyObject);
```

Comment:
```
source/dub/recipe/json.d(258,8): Error: no property require for type Json[string]
```

tl;dr: you can't use `require` here as it was only recently added and dub is supposed to support compilation for many D compilers.